chef/cookbooks/hyperv/attributes/default.rb

```diff
@@ -53,24 +53,24 @@
 default[:openstack][:location] = 'C:\OpenStack'
 
 default[:openstack][:nova][:name] = "nova"
-default[:openstack][:nova][:version] = "2014.2.1"
-default[:openstack][:nova][:file] = "nova-2014.2.1.tar.gz"
+default[:openstack][:nova][:version] = "12.0.0.0rc2"
+default[:openstack][:nova][:file] = "nova-stable-liberty.tar.gz"
```

Comment:
I'd feel more comfortable if we take a specific release (12.0.0.0) instead of the stable-liberty tarball. The reason is that we don't want to link to a random version from git.

We will still use the stable-liberty bits in the package; that's the "weird" bit where we "patch" the tarballs to use the version.

(same for the neutron & ceilometer tarballs)